package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=20"
   },
-  "version": "8.2.6",
+  "version": "8.2.7",
```

Comment:
please be careful here:
- a few arsenal PRs in progres, should only bump "right before" merging
- do we need a version for every arsenal PR, or can/should we "mutualize" and release once the last PR in the batch is merged?

Reply:
> do we need a version for every arsenal PR, or can/should we "mutualize" and release once the last PR in the batch is merged?

Should probably just bump at the last one unless we know the review cycle will take time.